website/src/content/api-reference/custom-webpack-config.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: "Custom Webpack Config"
+---
+
+By default Custom Applications include a webpack configuration to handle most of the common features like code transformation via Babel, loading CSS modules, loading images and SVGs, loading GraphQL documents, etc.
+However, it could be that your application requires some extra functionality like loading a particular file type, using a specific webpack plugin or include other source files in your repository.
+To extend the functionalities of webpack, you can define specific files in the root of your Custom Application project.
```

Comment:
I'm missing break lines in all the document so the formatting when rendering is a little bloated.

![image](https://user-images.githubusercontent.com/97907068/181748398-23f593f5-b371-4c6f-97bc-783f6c9c7daf.png)

Reply:
Updated here [58556a2](https://github.com/commercetools/merchant-center-application-kit/pull/2726/commits/58556a221343fe9e085103183132b592279fe0bc)

Reply:
I'm missing the break lines in the rest of this document's sections.
Also, it seems having an empty line is enough to add the breakline (no need for the `<br/>`.

You can take a look [here](https://github.com/commercetools/merchant-center-application-kit/blob/main/website/src/content/api-reference/create-mc-app.mdx?plain=1), for example.

![image](https://user-images.githubusercontent.com/97907068/182164549-e78dc742-d3cf-4ef3-aaad-0c354c7ced04.png)

Reply:
Updated here [7b17711](https://github.com/commercetools/merchant-center-application-kit/pull/2726/commits/7b17711077cc2936af454c93eabc2fe3c4462c5c)